src/EnergyPlus/DXCoils.cc

```diff
@@ -14656,9 +14656,6 @@ void ReportDXCoil(EnergyPlusData &state, int const DXCoilNum) // number of the c
     // Fills some of the report variables for the DX coils
 
     // Using/Aliasing
-    auto &DXElecCoolingPower = state.dataHVACGlobal->DXElecCoolingPower;
-    auto &DXElecHeatingPower = state.dataHVACGlobal->DXElecHeatingPower;
-    auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
```

Comment:
See [this wiki discussion](https://github.com/NREL/EnergyPlus/wiki/Local-Shortcut-Variables:-When-to-Use-a-Value-Copy-and-When-to-Use-a-Reference). At least 2 of these were removed. This should actually be Real64 TimeStepSys = state.dataHVACGlobal->TimeStepSys. Removal of the 1st 2 lines is OK, the last actually slow things down.

Reply:
If it is calculated only once in a given function you don't need to define a local variable `Real64 TimeStepSys` as shown below (this is how it was replaced in this fix):

`ReportingConstant = state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;`

If I understand your statement above correctly, you are saying that it is efficient to do it as follows:

```
Real64 TimeStepSys = state.dataHVACGlobal->TimeStepSys;
ReportingConstant = TimeStepSys * DataGlobalConstants::SecInHour;
```
The second method is a two step. Why not just do it as a one step calculation as in the first method?

Reply:
OK, your right. If it's only used once or twice then it is better this way.